.github/workflows/models-deploy.yml

```diff
@@ -1,10 +1,10 @@
 name: Sync Tensorflow models to Proxmox VM
-on: 
+on:
   push:
     branches:
       - master
       - deploy-*
-    tags: 
+    tags:
       - v*.*.*
```

Comment:
we have to change this to deploy on specific tags. see https://github.com/openfoodfacts/robotoff/issues/565